phalcon/Forms/Form.zep

```diff
@@ -78,6 +78,11 @@ class Form extends Injectable implements Countable, Iterator, AttributesInterfac
      */
     protected validation = null { set, get };
```

Comment:
Remove the `set` here, implement it in the method and return `this` so that it can be chained.